### ISSUE TITLE
Are You Going To Add Rag To The Capability?

### ISSUE THREAD

**Issue by profmarkhutchinson** [SystemSculpt/obsidian-systemsculpt-ai#46]
## Describe the question
Are You Considering Moving To A RAG Like Interaction With The Materials In The Vault, Rather Than Needing To Specify What Note Or Content Specifically Is Interacted With? Perhaps Even Exploring Ideas Across Multiple Notes?

**Comment by SystemSculpt** [comment]
Yes, this is my next big piece of the project after I have flawlessly implemented PDF/Doc/PowerPoint integration natively within the plugin. RAG, along with just auto-picking the best files based on the conversation topic/goal of the chat is my next focus with embeddings and such throughout your local vault.

Keeping it open until I've implemented a fully featured RAG system through embeddings.

**Comment by slockers37** [comment]
Having the option add folders and tags to context not just files would be awesome. Also if the user could ask the AI to pull up files from last week or specifying a date range or month and so on in natural language and the AI adds those to context.

**Comment by SystemSculpt** [comment]
[Automated] Closing in preparation of version 1.0 beta release. If this issue persists after updating, please reopen with details.
